test/metric_value_benchmark/disk_test.go

```diff
@@ -15,34 +16,36 @@ import (
 )
 
 type DiskTestRunner struct {
-	BaseTestRunner
+	test_runner.BaseTestRunner
 }
 
-func (t *DiskTestRunner) validate() status.TestGroupResult {
-	metricsToFetch := t.getMeasuredMetrics()
+var _ test_runner.ITestRunner = (*DiskTestRunner)(nil)
+
+func (t *DiskTestRunner) Validate() status.TestGroupResult {
+	metricsToFetch := t.GetMeasuredMetrics()
 	testResults := make([]status.TestResult, len(metricsToFetch))
 	for i, metricName := range metricsToFetch {
-		testResults[i] = t.validateDiskMetric(metricName)
+		testResults[i] = t.ValidateDiskMetric(metricName)
 	}
 
 	return status.TestGroupResult{
-		Name:        t.getTestName(),
+		Name:        t.GetTestName(),
 		TestResults: testResults,
 	}
 }
 
-func (t *DiskTestRunner) getTestName() string {
+func (t *DiskTestRunner) GetTestName() string {
 	return "Disk"
 }
 
-func (t *DiskTestRunner) getAgentConfigFileName() string {
+func (t *DiskTestRunner) GetAgentConfigFileName() string {
 	return "disk_config.json"
 }
-func (t *DiskTestRunner) getAgentRunDuration() time.Duration {
-	return minimumAgentRuntime
+func (t *DiskTestRunner) GetAgentRunDuration() time.Duration {
```

Comment:
Every other sanity test shares the same `GetAgentRunDuration` and also the `MinimumAgentRuntime`. Therefore, would it be better to add GetAgentRunDuration() in `BaseTestRunner`?

Reply:
Yeah I would second this nit pick